src/pxl_scripts/px/differential_flamegraph/differential.pxl

```diff
@@ -0,0 +1,77 @@
+# Copyright 2018- The Pixie Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import px
+
+negate = False
+# TODO(ddelnano): negation requires switching the type of join from right to left
+# or returning a different DataFrame. This might not be possible with pxl's current
+# functionality, but this should be implemented once it's possible.
```

Comment:
Ideally the `negate` parameter would be a script variable. In order to compute the negated flamegraph properly, the `how` on line 26 must change from right to left in addition to changing the `groupby` on line 35 to be against the `pod1` dataframe.

I tried to implement this by calling `merge_and_compute_delta` with swapped parameters or to use `px.select` to pick the correct join type (right vs left). Neither of these seemed to work. I think we might need some additional pxl support in order to accomplish this, but please let me know if you have any ideas.